<?xml  version="1.0"  encoding="UTF-8"?>
<ui  version="4.0">
  <class>Form</class>
  <widget  class="QWidget"  name="Form">
    <property  name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>500</height>
      </rect>
    </property>
    <property  name="windowTitle">
      <string>Form</string>
    </property>
    
    <layout  class="QVBoxLayout"  name="vlMain">
      <item>
        
        <widget class="QWidget" name="verticalLayoutWidget">
    
    
            <layout  class="QHBoxLayout"  name="horizontalLayout">
            <item>
                <widget  class="QSplitter"  name="splitter">
                <property  name="orientation">
                    <enum>Qt::Horizontal</enum>
                </property>
                
                <widget  class="QWidget"  name="wViewer">
                    <property  name="sizePolicy">
                    <sizepolicy  hsizetype="Preferred"  vsizetype="Preferred">
                        <horstretch>8</horstretch>
                        <verstretch>0</verstretch>
                    </sizepolicy>
                    </property>
                    <layout  class="QVBoxLayout"  name="vlViewer">
                    <item>
                        <widget  class="QMdiArea"  name="mdiArea"/>
                    </item>
                    </layout>
                </widget>
                
                <widget  class="QWidget"  name="wOptions">
                    <property  name="sizePolicy">
                    <sizepolicy  hsizetype="Minimum"  vsizetype="Minimum">
                        <horstretch>1</horstretch>
                        <verstretch>0</verstretch>
                    </sizepolicy>
                    </property>
                
                    <layout  class="QVBoxLayout"  name="vlOptions">

                    <item>
                        <widget  class="QFrame"  name="wActiveDset">
                        <property  name="frameShape">
                            <enum>QFrame::StyledPanel</enum>
                        </property>
                        <property  name="frameShadow">
                            <enum>QFrame::Raised</enum>
                        </property>
                        <property  name="lineWidth">
                            <number>8</number>
                        </property>

                        <layout  class="QVBoxLayout"  name="vlActiveDset">
                            
                            <item>
                            <widget  class="QLabel"  name="label_activeDset">
                                <property  name="text">
                                <string>Active Dataset</string>
                                </property>
                            </widget>
                            </item>

                            <item>
                            <layout  class="QHBoxLayout"  name="horizontalLayout_5">
                                <item>
                                <spacer  name="horizontalSpacer_5">
                                    <property  name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                    </property>
                                    
                                    <property name="sizeType">
                                        <enum>QSizePolicy::Fixed</enum>
                                    </property>
                                    
                                    <property  name="sizeHint"  stdset="0">
                                    <size>
                                        <width>30</width>
                                        <height>20</height>
                                    </size>
                                    </property>
                                </spacer>
                                </item>
                                <item>
                                    <widget  class="QLineEdit"  name="edit_activeDset">
                                        <property  name="text">
                                        <string> </string>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                            </item>

                        </layout>
                        </widget>
                    </item>
                    
                    <item>
                        <widget  class="QFrame"  name="wAction">
                        <property  name="frameShape">
                            <enum>QFrame::StyledPanel</enum>
                        </property>
                        <property  name="frameShadow">
                            <enum>QFrame::Raised</enum>
                        </property>
                        <property  name="lineWidth">
                            <number>8</number>
                        </property>
                        
                        <layout  class="QVBoxLayout"  name="vlTmp">
                            <item>
                            <widget  class="QLabel"  name="label_select">
                                <property  name="text">
                                <string>Action</string>
                                </property>
                            </widget>
                            </item>
                            
                            <item>
                            <layout  class="QHBoxLayout"  name="horizontalLayout_5">
                                <item>
                                <spacer  name="horizontalSpacer_5">
                                    <property  name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                    </property>
                                    
                                    <property name="sizeType">
                                        <enum>QSizePolicy::Fixed</enum>
                                    </property>
                                    
                                    <property  name="sizeHint"  stdset="0">
                                    <size>
                                        <width>30</width>
                                        <height>20</height>
                                    </size>
                                    </property>
                                </spacer>
                                </item>
                                <item>
                                    <layout  class="QHBoxLayout"  name="vlAction"/>
                                </item>
                                
                            </layout>
                            </item>
                            
                            <item>
                                <layout  class="QHBoxLayout"  name="hlTmp">

                                    <item>
                                        <spacer  name="horizontalSpacer_2">
                                            <property  name="orientation">
                                            <enum>Qt::Horizontal</enum>
                                            </property>
                                            
                                            <property name="sizeType">
                                                <enum>QSizePolicy::Fixed</enum>
                                            </property>
                                            
                                            <property  name="sizeHint"  stdset="0">
                                            <size>
                                                <width>30</width>
                                                <height>20</height>
                                            </size>
                                            </property>
                                        </spacer>
                                    </item>

                                    <item>
                                        <widget  class="QCheckBox"  name="check_createnew">
                                            <property  name="text">
                                            <string>Create New Dataset</string>
                                            </property>
                                        </widget>
                                    </item>
                                    
                                </layout>
                            </item>
                            
                        </layout>
                        
                        </widget>
                    </item>                    
                    
                    <item>
                        <widget  class="QFrame"  name="wOutVars">
                        <property  name="frameShape">
                            <enum>QFrame::StyledPanel</enum>
                        </property>
                        <property  name="frameShadow">
                            <enum>QFrame::Raised</enum>
                        </property>
                        <property  name="lineWidth">
                            <number>8</number>
                        </property>
                        
                        <layout  class="QVBoxLayout"  name="vlOutVars">
                            
                            <item>
                            <widget  class="QLabel"  name="label_select">
                                <property  name="text">
                                <string>Primary Key Var</string>
                                </property>
                            </widget>
                            </item>
                            
                            <item>
                            <layout  class="QHBoxLayout"  name="horizontalLayout_5">
                                <item>
                                <spacer  name="horizontalSpacer_5">
                                    <property  name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                    </property>
                                    
                                    <property name="sizeType">
                                        <enum>QSizePolicy::Fixed</enum>
                                    </property>
                                    
                                </spacer>
                                </item>
                                <item>
                                    <layout  class="QVBoxLayout"  name="vlPrimaryKeyVar"/>
                                </item>
                            </layout>
                            </item>
                            
                            <item>
                            <widget  class="QLabel"  name="label_select">
                                <property  name="text">
                                <string>Target Variables</string>
                                </property>
                            </widget>
                            </item>
                            
                            <item>
                            <layout  class="QHBoxLayout"  name="horizontalLayout_5">
                                <item>
                                <spacer  name="horizontalSpacer_5">
                                    <property  name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                    </property>
                                    
                                    <property name="sizeType">
                                        <enum>QSizePolicy::Fixed</enum>
                                    </property>
                                    
                                    <property  name="sizeHint"  stdset="0">
                                    <size>
                                        <width>30</width>
                                        <height>20</height>
                                    </size>
                                    </property>
                                </spacer>
                                </item>
                                <item>
                                <layout  class="QVBoxLayout"  name="vlComboOut"/>
                                </item>
                            </layout>
                            </item>

                            <item>
                            <widget  class="QLabel"  name="label_select2">
                                <property  name="text">
                                <string>Out suffix: </string>
                                </property>
                            </widget>
                            </item>
                            
                            <item>
                            <layout  class="QHBoxLayout"  name="horizontalLayout_6">
                                <item>
                                <spacer  name="horizontalSpacer_6">
                                    <property  name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                    </property>
                                    
                                    <property name="sizeType">
                                        <enum>QSizePolicy::Fixed</enum>
                                    </property>
                                    
                                    <property  name="sizeHint"  stdset="0">
                                    <size>
                                        <width>30</width>
                                        <height>20</height>
                                    </size>
                                    </property>
                                </spacer>
                                </item>
                                <item>
                                    <widget  class="QLineEdit"  name="edit_out_suff">
                                    </widget>
                                </item>
                            </layout>
                            </item>
                            
                        </layout>
                        </widget>
                    </item>
                    
                    
                    <item>
                        <widget  class="QFrame"  name="wNormVars">
                        
                        <property  name="frameShape">
                            <enum>QFrame::StyledPanel</enum>
                        </property>
                        <property  name="frameShadow">
                            <enum>QFrame::Raised</enum>
                        </property>
                        <property  name="lineWidth">
                            <number>8</number>
                        </property>
                        
                        <layout  class="QVBoxLayout"  name="vlNormalize">

                            <item>
                            <widget  class="QLabel"  name="label_select">
                                <property  name="text">
                                <string>Divide by</string>
                                </property>
                            </widget>
                            </item>
                            
                            <item>
                            <layout  class="QHBoxLayout"  name="hlDivideBy">
                                <item>
                                <spacer  name="horizontalSpacer_5">
                                    <property  name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                    </property>
                                    
                                    <property name="sizeType">
                                        <enum>QSizePolicy::Fixed</enum>
                                    </property>                                    
                                    
                                    <property  name="sizeHint"  stdset="0">
                                    <size>
                                        <width>30</width>
                                        <height>20</height>
                                    </size>
                                    </property>
                                </spacer>
                                </item>
                                <item>
                                <layout  class="QVBoxLayout"  name="vlComboNormVar"/>
                                </item>
                            </layout>
                            </item>
                            
                            <item>
                            <layout  class="QHBoxLayout"  name="horizontalLayout_8">

                            <item>
                                <spacer  name="horizontalSpacer_8">
                                <property  name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property  name="sizeHint"  stdset="0">
                                    <size>
                                    <width>40</width>
                                    <height>20</height>
                                    </size>
                                </property>
                                </spacer>
                            
                            
                            </item>
                            
                            <item>
                                <widget  class="QPushButton"  name="normalizeBtn">
                                <property  name="text">
                                    <string>Normalize</string>
                                </property>
                                </widget>
                            </item>
                            
                            </layout>
                            </item>
                            
                        </layout>
                        </widget>
                    </item>
                    
                    <item>
                        <widget  class="QFrame"  name="wAdjustVars">
                        <property  name="frameShape">
                            <enum>QFrame::StyledPanel</enum>
                        </property>
                        <property  name="frameShadow">
                            <enum>QFrame::Raised</enum>
                        </property>
                        <property  name="lineWidth">
                            <number>8</number>
                        </property>
                        
                        <layout  class="QVBoxLayout"  name="vlCov">

                            <item>
                            <widget  class="QLabel"  name="label_select">
                                <property  name="text">
                                <string>Covariates to keep</string>
                                </property>
                            </widget>
                            </item>

                            <item>
                            <layout  class="QHBoxLayout"  name="horizontalLayout_5">
                                <item>
                                <spacer  name="horizontalSpacer_5">
                                    <property  name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                    </property>
                                    
                                    <property name="sizeType">
                                        <enum>QSizePolicy::Fixed</enum>
                                    </property>
                                    
                                    <property  name="sizeHint"  stdset="0">
                                    <size>
                                        <width>30</width>
                                        <height>20</height>
                                    </size>
                                    </property>
                                </spacer>
                                </item>
                                <item>
                                <layout  class="QVBoxLayout"  name="vlComboCovKeep"/>
                                </item>
                            </layout>
                            </item>

                            <item>
                            <widget  class="QLabel"  name="label_select">
                                <property  name="text">
                                <string>Covariates to correct</string>
                                </property>
                            </widget>
                            </item>
                            
                            <item>
                            <layout  class="QHBoxLayout"  name="horizontalLayout_5">
                                <item>
                                <spacer  name="horizontalSpacer_5">
                                    <property  name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                    </property>
                                    
                                    <property name="sizeType">
                                        <enum>QSizePolicy::Fixed</enum>
                                    </property>
                                    
                                    <property  name="sizeHint"  stdset="0">
                                    <size>
                                        <width>30</width>
                                        <height>20</height>
                                    </size>
                                    </property>
                                </spacer>
                                </item>
                                <item>
                                <layout  class="QVBoxLayout"  name="vlComboCovCorr"/>
                                </item>
                            </layout>
                            </item>

                            <item>
                            <widget  class="QLabel"  name="label_select2">
                                <property  name="text">
                                <string>Estimation Sample</string>
                                </property>
                            </widget>
                            </item>

                            <item>
                            <layout  class="QHBoxLayout"  name="horizontalLayout_6">
                                <item>
                                <spacer  name="horizontalSpacer_6">
                                    <property  name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                    </property>
                                    
                                    <property name="sizeType">
                                        <enum>QSizePolicy::Fixed</enum>
                                    </property>
                                    
                                    <property  name="sizeHint"  stdset="0">
                                    <size>
                                        <width>30</width>
                                        <height>20</height>
                                    </size>
                                    </property>
                                </spacer>
                                </item>
                                <item>
                                <layout  class="QVBoxLayout"  name="vlComboSel"/>
                                </item>
                            </layout>
                            </item>

                            <item>
                            <layout  class="QHBoxLayout"  name="horizontalLayout_8">

                            <item>
                                <spacer  name="horizontalSpacer_8">
                                <property  name="orientation">
                                    <enum>Qt::Horizontal</enum>
                                </property>
                                <property  name="sizeHint"  stdset="0">
                                    <size>
                                    <width>40</width>
                                    <height>20</height>
                                    </size>
                                </property>
                                </spacer>
                            
                            
                            </item>
                            
                            <item>
                                <widget  class="QPushButton"  name="adjustBtn">
                                <property  name="text">
                                    <string>Adjust</string>
                                </property>
                                </widget>
                            </item>
                            
                            </layout>
                            </item>
                        </layout>
                            
                        </widget>
                    </item>
                                        
                    
                    <item>
                        <spacer  name="verticalSpacer">
                        <property  name="orientation">
                            <enum>Qt::Vertical</enum>
                        </property>
                        <property  name="sizeHint"  stdset="0">
                            <size>
                            <width>20</width>
                            <height>40</height>
                            </size>
                        </property>
                        </spacer>
                    </item>
                    
                    </layout>
                </widget>
                </widget>
            </item>
            </layout>
            

        </widget>
      </item>

      </layout>

            
  </widget>
  <resources/>
  <connections/>
</ui>
